HW1/P6/P6.py

```diff
@@ -0,0 +1,78 @@
+import pyspark
+
+# filter function
+def filter_out_Capital(word):
+    wlen=len(word)
+    count=0
+    for l in word:
+        if l in 'ABCDEFGHIJKLMNOPQRSTUVWXYZ':
```

Comment:
Once more: clear code!
This can also be done with a quick comparison: `word.upper() == word`.